hbase-common/src/main/java/org/apache/hadoop/hbase/HConstants.java

```diff
@@ -972,6 +972,17 @@
   /** Whether nonces are enabled; default is true. */
   public static final String HBASE_RS_NONCES_ENABLED = "hbase.regionserver.nonces.enabled";
 
+  /** Some configuration param about multiget in parallel */
+  public static final String HBASE_RS_PARALLEL_GET_ENABLED =
+      "hbase.regionserver.parallel.get.enabled";
+  public static final boolean DEFAULT_PARALLEL_GET_ENABLED = false;
```

Comment:
Why not enable it by default? What would be the downside?

Reply:
Each multi request will be handle by one Handler thread, if there are 10 multi request, the concurrency is 10 before enable this feature. But if we set the parGet threadpool’s capacity to 5 and enable this feature, the concurrency will be reduced to 5, will be some performance loss. That's what I thought before.